.chloggen/dinesh.gurumurthy_expand-resource-attributes.yaml

```
@@ -0,0 +1,23 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadog/connetor
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable connetor to use any attribute from the resource as Container Tag
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [32224]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  This change allows the connector to use any attribute from the resource as a container tag. This is useful when you want to use a custom attribute from the resource as a container tag. For example, you can use the `namespace` attribute from the resource as a container tag. This change is backward compatible and does not require any changes to the existing configuration.
```

The last part is unnecessary: if it were a breaking change then the `change_type` would be `breaking`, not `enhancement`.
```suggestion
  This change allows the connector to use any attribute from the resource as a container tag. This is useful when you want to use a custom attribute from the resource as a container tag. For example, you can use the `namespace` attribute from the resource as a container tag.
```